src/main/java/page_objects/LoginPage.java

```diff
@@ -0,0 +1,42 @@
+package page_objects;
+
+import helpers.DriverHelper;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+
+public class LoginPage extends BasePage {
+
```

Comment:
BasePage doesn't have this line
Please remove it or add it on BasePage to make they the same format

Reply:
i fixed it. Could you help me check it again, thanks so much